opencost/src/settings.tsx

```diff
@@ -0,0 +1,85 @@
+import { NameValueTable } from '@kinvolk/headlamp-plugin/lib/components/common';
+import Box from '@mui/material/Box';
+import Checkbox from '@mui/material/Checkbox';
+import MenuItem from '@mui/material/MenuItem';
+import Select from '@mui/material/Select';
+import TextField from '@mui/material/TextField';
+
+export function Settings(props) {
+  const { data, onDataChange } = props;
+
+  const settingsRows = [
+    {
+      name: 'Enable in list view',
```

Comment:
What does this mean?
I noticed there was a column in the namespaces list view for open cost even if this shows as disabled by default.

Reply:
Yes it is to show/hide the opencost in the list view. I have fixed the default value for this setting,